src/toolbelt/core.cljc

```diff
@@ -30,12 +31,13 @@
   ([s]
    (cond
      (number? s) s
-     (map? s)    (throw (ex-info "map arguments require keys" {:map s}))
+     (map? s) (throw (ex-info "map arguments require keys" {:map s}))
      (string? s) (when-not (string/blank? s)
                    (let [s (re-find #"-?\d+" s)]
-                     (Long. s)))
-     :otherwise  (throw (ex-info (str "cannot convert argument of type " (type s))
-                                 {:argument s}))))
+                     #?(:clj  (Long. s)
+                        :cljs (js/parseInt s))))
+     :otherwise (throw (ex-info (str "cannot convert argument of type " (type s))
+                                {:argument s}))))
```

Comment:
Do we have style guidance on this? I prefer the conditions to line up.

Reply:
We do prefer that they line up.